packages/common/src/components/Footer.jsx

```diff
@@ -7,23 +7,51 @@
  */
 
 import React from 'react'
+import { FooterLink } from '@kernel/common'
 
+const env = process.env.REACT_APP_DEPLOY_TARGET || 'PROD'
+const prefix = env === 'STAGING' ? 'staging.' : ''
 const defaults = {
   backgroundColor: 'bg-gray-900',
-  textColor: 'text-white'
+  textColor: 'text-white',
+  footerLinks: [
+    { href: 'https://' + prefix + 'explore.kernel.community', title: 'explorer' },
+    { href: 'https://' + prefix + 'unprofile.kernel.community', title: 'unprofile' },
+    { href: 'https://' + prefix + 'adventures.kernel.community', title: 'adventures' },
+    { href: 'https://' + prefix + 'wallet.kernel.community', title: 'portal' },
+  ]
 }
 
-export default function Footer (props) {
+export default function Footer(props) {
```

Comment:
Just some cleanup, not a comment on your changes but, does something like this work?:
```
export default function Footer({ children, backgroundColor = defaults.backgroundColor, textColor = defaults.textColor }) {
```
If it works it might be cleaner but I don't remember whether the default can be an expression like that..

Reply:
Very cool point!
Changed the function param to include defaults and cleaned up the function itself.